src/utils/resources/bootableresources/hooks/useBootableVolumes.ts

```diff
@@ -10,16 +9,22 @@ import {
 import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
 import { IoK8sApiCoreV1PersistentVolumeClaim } from '@kubevirt-ui/kubevirt-api/kubernetes';
 import { isEqualObject } from '@kubevirt-utils/components/NodeSelectorModal/utils/helpers';
+import { BootableVolume } from '@kubevirt-utils/resources/bootableresources/types';
 import {
   convertResourceArrayToMap,
   getReadyOrCloningOrUploadingDataSources,
 } from '@kubevirt-utils/resources/shared';
 import { isEmpty } from '@kubevirt-utils/utils/utils';
 import { Operator, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
 
-import { UseBootableVolumesValues } from '../utils/types';
-
-type UseBootableVolumes = (namespace?: string) => UseBootableVolumesValues;
+type UseBootableVolumes = (namespace?: string) => {
+  bootableVolumes: BootableVolume[];
+  loaded: boolean;
+  loadError?: any;
```

Comment:
why ? on loadError? can you also use Error type instead of any?

Reply:
`useK8sWatchResource` use `any` for the error and i think because it's something similar to `Error` but its not exactly that. Seems like have something more than Error